spec/factories/accessories.rb

```diff
@@ -7,6 +7,20 @@
     after(:create) do |instrument, evaluator|
       instrument.accessories << evaluator.accessory
     end
+
+    trait :not_reservation_only do
```

Comment:
And can we call this trait something non-negated? Like `timer` or `timer_without_relay`?